src/components/counter/Counter.tsx

```diff
@@ -38,7 +43,7 @@ export class Counter extends React.Component<Props>{
   render(){
     return (
       <CounterDiv {...this.props} onClick={this.props.onClick}>
-        <span>{this.props.clicked}</span>
+        <SpanStyled>{this.props.clicked}</SpanStyled>
```

Comment:
Je kan de styling ook in het CounterDiv styled component stoppen. Op die manier moet je minder styled components maken. 

```
const CounterDivStyled = styled.div`
...<insert alle stijlen hier>
> *:first-child {
padding-right: var(--spacing-m);
}
`
```